lib/recipes-data/src/lib/fastly.ts

```diff
@@ -35,21 +35,24 @@ function removeLeadingAndTrailingSlash(from: string): string {
  * @param apiKey Fastly API key to authenticate the request.  The function will throw if this is undefined or empty.
  * @param purgeType Whether to execute a soft or hard purge (default soft). See the docs on PurgeType for more information.
  */
-export async function sendFastlyPurgeRequest(
-	contentPath: string,
-	apiKey: string,
-	purgeType?: PurgeType,
-) {
-	if (!ContentPrefix) {
-		throw new Error('Cannot purge because CONTENT_URL_BASE is not set');
-	}
+export async function sendFastlyPurgeRequest({
+	contentPath,
+	apiKey,
+	contentPrefix,
+	purgeType,
+}: {
+	contentPath: string;
+	apiKey: string;
+	contentPrefix: string;
```

Comment:
should probably update the docs here too 😁

Reply:
👍 4185046f4c5b70dc0f8fdde462a26c754484cd7f